codalab/apps/teams/forms.py

```diff
@@ -89,3 +95,144 @@ def clean(self):
         cleaned_data = super(TeamMembershipForm, self).clean()
 
         return cleaned_data
+
+
+class OrganizerTeamForm(forms.ModelForm):
+
+    text_members = forms.CharField(required=False, widget=forms.Textarea)
+
+    class Meta:
+        model = Team
+        fields = ('description', 'name')
+
+    def __init__(self, *args, **kwargs):
+        self.user_list = []
+        self.competition = Competition.objects.get(pk=kwargs.pop('competition_pk'))
+        self.creator = ClUser.objects.get(pk=kwargs.pop('creator_pk'))
+        super(OrganizerTeamForm, self).__init__(*args, **kwargs)
+        # This check is to see if we have an existing object with a PK, or if we're making a new one
+        # If we have an existing we grab existing members for the form.
+        if hasattr(self, 'instance'):
+            if hasattr(self.instance, 'pk'):
+                if self.instance and self.instance.pk:
+                    team = Team.objects.get(pk=self.instance.pk)
+                    current_members = team.members.all()
```

Comment:
If you want the names just do `team.members.all().values_list('username', flat=True)`